README.md

```diff
@@ -1,15 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+  LEADERBOARD
 
 > Description the project.
-
+This is a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. It develop as a first working version of the app following a wireframe, but without styling - and it is only focus on functionality. 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
+- Webpack
 
```

Comment:
- [ ] In the README file, Please add steps on how to clone and run the application.

Reply:
@thecodechaser,

Good afternoon. Hope your day is good. What you are saying is for day 2 which will be needed creating another game and save.  Pls can you really put me through on this. 
![image](https://user-images.githubusercontent.com/101729024/186187541-7a3d5814-09c4-4841-a165-c2d34832f6dc.png)
Thank you.

Reply:
@ecekpo Hi, Add steps how to clone and install webpack dependencies and run the development server.. hint: npm install, npm start

Reply:
@ecekpo Please ask for another code review since I was just following along with the first reviewer requirements, we apologize :neutral_face: